packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -61,7 +61,10 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       respOnlySdp: true,
       allowMultiDevice: true,
       ensureConversation: ensureConversation || false,
-      supportsNativeLobby: 1
+      supportsNativeLobby: 1,
+      clientMediaPreferences: {
+        preferTranscoding: preferTranscoding || true
```

Comment:
```suggestion
        preferTranscoding: preferTranscoding ?? true
```

Reply:
What does that do!?

Reply:
Instead of testing for `falsey`, it tests for `undefined` or `null`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator